deepcave/plugins/hyperparameter/importances.py

```diff
@@ -243,8 +264,8 @@ def load_dependency_inputs(self, run, _: Any, inputs: Dict[str, Any]) -> Dict[st
         n_hps = inputs["n_hps"]["value"]
 
         # Pre-set values
-        if objective_value is None:
-            objective_value = objective_ids[0]
```

Comment:
I think moving the next two lines above

`        objective_options = get_select_options(objective_names, objective_ids)
        objective_options2 = [dic for dic in objective_options if dic['value'] != objective_value1] # make sure the same objective cannot be chosen twice
        objective_options2 +=  [{'label': 'Select objective ...', 'value': -1}] # add the option to deselect the second objective
`

should ensure that objective_options2 is updated right after choosing objective_options2